.github/workflows/daily.yml

```diff
@@ -371,106 +371,61 @@ jobs:
+          echo "skiptests: ${{github.event.inputs.skiptests}}"
+          echo "test_args: ${{github.event.inputs.test_args}}"
+          echo "cluster_test_args: ${{github.event.inputs.cluster_test_args}}"
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        with:
+          repository: ${{ env.GITHUB_REPOSITORY }}
+          ref: ${{ env.GITHUB_HEAD_REF }}
+      - name: make
+        run: |
+          make SERVER_CFLAGS='-Werror'
+      - name: testprep
+        run: "sudo apt-get install vmtouch\nmkdir /tmp/master \nmkdir /tmp/slave\n"
+      - name: warm up
+        run: |
+          ./src/valkey-server --daemonize yes --logfile /dev/null
+          ./src/valkey-benchmark -n 1 > /dev/null
+          ./src/valkey-cli save | grep OK > /dev/null
+          vmtouch -v ./dump.rdb > /dev/null
+      - name: test
+        run: "echo \"test SAVE doesn't increase cache\"\nCACHE0=$(grep -w file /sys/fs/cgroup/memory.stat | awk '{print $2}')\necho \"$CACHE0\"\n./src/valkey-server --daemonize yes --logfile /dev/null --dir /tmp/master --port 8080 --repl-diskless-sync no --pidfile /tmp/master/valkey.pid --rdbcompression no --enable-debug-command yes\n./src/valkey-cli -p 8080 debug populate 10000 k 102400\n./src/valkey-server --daemonize yes --logfile /dev/null --dir /tmp/slave --port 8081 --repl-diskless-load disabled --rdbcompression no\n./src/valkey-cli -p 8080 save > /dev/null\nVMOUT=$(vmtouch -v /tmp/master/dump.rdb)\necho $VMOUT\ngrep -q \" 0%\" <<< $VMOUT \nCACHE=$(grep -w file /sys/fs/cgroup/memory.stat | awk '{print $2}')\necho \"$CACHE\"\nif [ \"$(( $CACHE-$CACHE0 ))\" -gt \"8000000\" ]; then exit 1; fi\n#magic___^_^___line\necho \"test replication doesn't increase cache\"\n./src/valkey-cli -p 8081 REPLICAOF 127.0.0.1 8080 > /dev/null\nwhile [ $(./src/valkey-cli -p 8081 info replication | grep \"master_link_status:down\") ]; do sleep 1; done;\nsleep 1 # wait for the completion of cache reclaim bio\nVMOUT=$(vmtouch -v /tmp/master/dump.rdb)\necho $VMOUT\ngrep -q \" 0%\" <<< $VMOUT \nVMOUT=$(vmtouch -v /tmp/slave/dump.rdb)\necho $VMOUT\ngrep -q \" 0%\" <<< $VMOUT \nCACHE=$(grep -w file /sys/fs/cgroup/memory.stat | awk '{print $2}')\necho \"$CACHE\"\nif [ \"$(( $CACHE-$CACHE0 ))\" -gt \"8000000\" ]; then exit 1; fi\n#magic___^_^___line\necho \"test reboot doesn't increase cache\"\nPID=$(cat /tmp/master/valkey.pid)\nkill -15 $PID\nwhile [ -x /proc/${PID} ]; do sleep 1; done\n./src/valkey-server --daemonize yes --logfile /dev/null --dir /tmp/master --port 8080\nwhile [ $(./src/valkey-cli -p 8080 info persistence | grep \"loading:1\") ]; do sleep 1; done;\nsleep 1 # wait for the completion of cache reclaim bio\nVMOUT=$(vmtouch -v /tmp/master/dump.rdb)\necho $VMOUT\ngrep -q \" 0%\" <<< $VMOUT\nCACHE=$(grep -w file /sys/fs/cgroup/memory.stat | awk '{print $2}')\necho \"$CACHE\"\nif [ \"$(( $CACHE-$CACHE0 ))\" -gt \"8000000\" ]; then exit 1; fi\n#magic___^_^___line\n"
```

Comment:
Is this the result of the yml format? it is a lot harder to read now.

Reply:
Yh. I'll include a line length limit

Reply:
This is caused by trailing whitespaces on some lines. See: https://github.com/google/yamlfmt/issues/185